src/contexts/boundary/ErrorBoundary.tsx

```diff
@@ -27,22 +26,30 @@ class ErrorBoundary extends Component {
       const errorMessage = formatError(this.state.error);
 
       return (
-        <Page centered topPadding>
+        <StyledErrorBoundary>
           <BaseXL>{errorMessage}</BaseXL>
           <Spacer height={48} />
           <StyledReachOut>
             If you&apos;re continuing to see this error, reach out to us on{' '}
             <GalleryLink href="https://discord.gg/QcJjCDucwK">Discord</GalleryLink>.
           </StyledReachOut>
           <Spacer height={16} />
-        </Page>
+        </StyledErrorBoundary>
       );
     }
 
     return this.props.children;
   }
 }
 
+const StyledErrorBoundary = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  height: 100vh;
+`;
```

Comment:
this will be a common appearance. without the `Page` component, views will now style themselves